### PR TITLE
[new][author]Added validatorjs@1.1.2

### PR DESCRIPTION
**Pull request by ppoffice**

### CODE REVIEW

File: ajax/libs/validatorjs/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "validation",
+    "laravel"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ppoffice/validator.js.git",
+    "basePath": "dist/",
+    "files": [
+     "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/validatorjs/package.json]: Missing one space here.

**Comment by PeterDaveHello** [comment]
Should not touch `sortable` here, please read our sparse-checkout.md to help you fix it.

**Comment by ppoffice** [comment]
@PeterDaveHello Thank you very much :blush:

**Comment by PeterDaveHello** [comment]
Would you please squash them into one as our document said.

**Comment by ppoffice** [comment]
@PeterDaveHello I think it's OK now.

**Comment by PeterDaveHello** [comment]
Can you also rebase your branch with our latest master branch? Commit's parent was a little bit too old, thanks.

**Comment by ppoffice** [comment]
It's all done. @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Thank you.

**Pull request merged**